content/en/users/check-in/voms/_index.md

```diff
@@ -252,34 +259,33 @@ existing VOs
 
 Every VO needs two different pieces of information:
 
--   the `vomses` configuration files, where the details of the VO are
-    stored (e.g. name, server, ports). These are stored by default at
-    `/etc/vomses` and are normally named following this convention:
-    `<vo name>.<server name>` (e.g. for fedcloud.egi.eu VO, you would
-    have `fedcloud.egi.eu.voms1.grid.cesnet.cz` and
-    `fedcloud.egi.eu.voms2.grid.cesnet.cz`.
--   the `.lsc` files that describe the trust chain of the VOMS server.
-    These are stored at `/etc/grid-security/vomsdir/<vo name>` and there
-    should be one file for each of the VOMS server of the VO.
-
-You can check specific configuration for your VO at the [Operations
-portal](https://operations-portal.egi.eu/vo). Normally each VOMS server
-has a *Configuration Info* link where the exact information to include
-in the `vomses` and `.lsc` files is shown.
+- the `vomses` configuration files, where the details of the VO are stored (e.g.
+  name, server, ports). These are stored by default at `/etc/vomses` and are
+  normally named following this convention: `<vo name>.<server name>` (e.g. for
+  fedcloud.egi.eu VO, you would have `fedcloud.egi.eu.voms1.grid.cesnet.cz` and
+  `fedcloud.egi.eu.voms2.grid.cesnet.cz`.
+- the `.lsc` files that describe the trust chain of the VOMS server. These are
+  stored at `/etc/grid-security/vomsdir/<vo name>` and there should be one file
+  for each of the VOMS server of the VO.
+
+You can check specific configuration for your VO at the
+[Operations portal](https://operations-portal.egi.eu/vo). Normally each VOMS
+server has a _Configuration Info_ link where the exact information to include in
+the `vomses` and `.lsc` files is shown.
 
 ### Proxy creation
 
-Once you have the VO information configured (`vomses` and `.lsc`) and
-your certificate available in your `$HOME/.globus` directory you can
-create a VOMS proxy to be used with clients with:
+Once you have the VO information configured (`vomses` and `.lsc`) and your
+certificate available in your `$HOME/.globus` directory you can create a VOMS
+proxy to be used with clients with:
 
-``` {.console}
+```{.console}
 voms-proxy-init --voms <name of the vo> --rfc
 ```
 
 See for example, using `fedcloud.egi.eu` VO:
 
-``` {.console}
+```{.console}
```

Comment:
Please use proper markdown fenced code markers (everywhere in the doc), like;
```
```sh
```

(sh is usually preferred over console as it highlights more stuff).